supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -85,7 +85,7 @@
 [discrete]
 [[active-directory-security-identifier]]
 ==== image:images/yes.png[yes] Active Directory security identifier (noun)
-*Description*: A security identifier (SID) is a unique ID number assigned to an object in Active Directory, such as a user, group, or host. A SID is the functional equivalent of UIDs and GIDs in Linux.
+*Description*: A _security identifier_ (SID) is a unique ID number assigned to an object in Active Directory, such as a user, group, or host. A SID is the functional equivalent of UIDs and GIDs in Linux.
```

Comment:
Might be out of scope of this PR, but wondering if this should say

```suggestion
*Description*: An _Active Directory security identifier_ (SID) is a unique ID number assigned to an object in Active Directory, such as a user, group, or host. A SID is the functional equivalent of UIDs and GIDs in Linux.
```

Reply:
:heavy_check_mark: